app/models/concerns/listable.rb

```diff
@@ -1,12 +1,14 @@
 module Listable
+  NUMBER_OF_PAST_EVENTS_FOR_INDEX = 30
```

Comment:
Hardcoding limits feels like it limits flexibility. For instance consider if we wanted to put a little box elsewhere on the site that shows the last three events, we'd need a new scope rather than being able to have
`Course.past_display(30)` on one  page and `Course.past_display(3)` on another. What do you think to having this as parameter that you pass into the scope rather than something hardcoded? 30 could still remain the default so `Course.past_display()` would still return 30 results